js/custom_projection.js

```diff
@@ -0,0 +1,147 @@
+// A modified d3.geo.albersUsa to include Puerto Rico.
+function albersUsaPr() {
+  var ε = 1e-6;
+
+  var lower48 = d3.geoAlbers();
+
+  // EPSG:3338
+  var alaska = d3.geoConicEqualArea()
+      .rotate([154, 0])
+      .center([-2, 58.5])
+      .parallels([55, 65]);
+
+  // ESRI:102007
+  var hawaii = d3.geoConicEqualArea()
+      .rotate([157, 0])
+      .center([-3, 19.9])
+      .parallels([8, 18]);
+
+  // XXX? You should check that this is a standard PR projection!
```

Comment:
who's "you" here? Is that something that still needs to be done?

Reply:
That was the author's comment, It projected PR just fine, so I don't think its anything to worry about.

Reply:
can you remove the comment?

Reply:
Sure